topydo.conf

```diff
@@ -6,6 +6,8 @@ default_command             = ls
 ; archive_filename          = done.txt
 colors                      = 1
 identifiers                 = linenumber ; or: text
+console_height              = 25
```

Comment:
[shutil.get_terminal_size()](https://docs.python.org/3/library/shutil.html#querying-the-size-of-the-output-terminal) seems like the best option, as it is part of the standard library, but is only available for Python 3.3+ (and not available for Python 2.7).

I think this is my preferred solution, but I'm not sure how best to deal with Python 2.7 and 3.2